driver/src/main.rs

```diff
@@ -91,12 +91,11 @@ struct Options {
     /// JSON encoded object of which tokens/orders to ignore.
     ///
     /// For example: '{
-    ///   "tokens": [1, 2],
+    ///   "tokens": {"Whitelist": [1, 2]},
```

Comment:
I guess it is not possible to provide an example that uses both a whitelist and a blacklisted token... since these are mutually exclusive.  (Perhaps we could add a comment here or somewhere stating that blacklist could be used in the same way.